src/MLJXGBoostInterface.jl

```diff
@@ -775,22 +280,29 @@ MMI.load_path(::Type{<:XGBoostRegressor}) = "$PKG.XGBoostRegressor"
 MMI.input_scitype(::Type{<:XGBoostRegressor}) = Table(Continuous)
 MMI.target_scitype(::Type{<:XGBoostRegressor}) = AbstractVector{Continuous}
 MMI.docstring(::Type{<:XGBoostRegressor}) =
-    "The XGBoost gradient boosting method, for use with "*
-    "`Continuous` univariate targets. "
+"The XGBoost gradient boosting method, for use with "*
```

Comment:
Please remove entirely the `docstring` trait declarations. These now fallback to the standard and detailed docstring which you have kindly added. (Otherwise, the full docstring will not be available in the model registry.)